pkg/sqlstore/impl/system/store.go

```diff
@@ -97,8 +97,8 @@ func (s *SystemStore) GetTablesByController(ctx context.Context, controller stri
 		return []sqlstore.Table{}, fmt.Errorf("sanitizing address: %s", err)
 	}
```

Comment:
Most of the changes in this file are to adjust for changes in the name of fields and their types. The names and types `sqlc` generated was not 100% compatible with our code. 

I played a lot with `sqlc` trying to generate the names and types compatible with existing code. But since our code evolved without `sqlc` we don't follow a consistent type schema for some fields. For example, block number is `int64` in some places and `uint64` in others. No matter what rule I put in `sqlc` in some place we have to do the casting.  The same happens with timestamp fields: `time.Time` and `int64`.

Reply:
:+1: